index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Palindrome Checker</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      text-align: center;
+      background: linear-gradient(90deg, white, black, red);
+      color: white;
+      margin: 0;
+      padding: 20px;
+      box-sizing: border-box;
+    }
+
+    h1 {
+      color: red;
+    }
+
+    label {
+      display: block;
+      font-size: 20px;
+      margin-bottom: 10px;
+    }
+
+    input[type="text"] {
+      width: 300px;
+      height: 30px;
+      font-size: 18px;
+      padding: 5px;
+    }
+
+    button {
+      background-color: red;
+      color: white;
+      font-size: 20px;
+      padding: 10px 20px;
+      border: none;
+      cursor: pointer;
+    }
+
+    button:hover {
+      background-color: darkred;
+    }
+
+    #result {
+      font-size: 24px;
+      margin-top: 20px;
+    }
+  </style>
+</head>
+<body>
+  <h1>Palindrome Checker</h1>
+  <label for="text-input">Enter a word or phrase:</label>
+  <input type="text" id="text-input">
+  <br><br>
+  <button id="check-btn">Check</button>
+  <br><br>
+  <div id="result"></div>
+
+  <script>
+    const checkButton = document.getElementById('check-btn');
+    const resultDiv = document.getElementById('result');
+
+    checkButton.addEventListener('click', function() {
+      const userInput = document.getElementById('text-input').value;
+
+      if (userInput.trim().length === 0) {
+        alert('Please input a value');
+      } else {
+        const cleanedInput = userInput.toLowerCase().replace(/[^a-z0-9]/g, '');
+        const reversedInput = cleanedInput.split('').reverse().join('');
+        
+        if (cleanedInput === reversedInput) {
+          resultDiv.innerText = userInput + ' is a palindrome';
+        } else {
+          resultDiv.innerText = userInput + ' is not a palindrome';
+        }
+      }
+    });
+  </script>
```

Comment:
- Same here you need to move your JS code to your `script.js` file and link your JS file using `<script>` tag at the end of your HTML body code.